### DISCUSSION TITLE
istoreos 似乎在不断请求 Packages.sig

### DISCUSSION THREAD

**Discussion by ZhiShengYuan** [istoreos/istoreos#1534] [General]
Hi 管理员/贡献者:
我近日在多个开源镜像站上观察到有ua为 "curl/8.4.0" 或者 "curl/8.6.0" 的客户端在大量请求 "uri":"/openwrt/releases/22.03.6/packages/*/base/Packages.sig",我通过搜索发现主流openwrt发行版中只有istoreos还在使用这一版本且使用了mirrors.cernet.edu.cn,我希望知道这是否是由于你们的配置出现了一些错误?这一文件的请求数量较为巨大，虽然整体传输量较低暂未对我们的镜像站构成较大的压力，但是我仍然希望知道这是否是由于错误的配置所导致的。
顺颂商祺

**Comment by ZhiShengYuan** [comment]
作为参考，在某一特定镜像站上的数据如下
118,455 (01.57%)(requests)	2,005 (00.55%)(ip)	/openwrt/releases/22.03.6/packages/x86_64/base/Packages.sig
在其余镜像站上观测到的log类似

**Comment by ks-bao** [comment]
应该不是代码错误。因为我们有个软件中心，当用户要安装插件的时候，如果短时间内没 opkg update，就会主动帮用户运行一次 opkg update，然后导致了 Packages.sig 的请求吧。我猜测是这样的原因。

**Reply by jjm2473** [reply]
在这里跟进了 https://github.com/istoreos/istoreos/issues/1541

**Reply by jjm2473** [reply]
跟istore的关系不大，如果是istore，不会只下载 Packages.sig。主要还是quickstart的网络检测。

**Comment by ZhiShengYuan** [comment]
问题已经在 https://github.com/istoreos/istoreos/issues/1541 得到了解决
